Fixed output default in unify command (#396)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

> * Fixed a bug when an output argument was not passed to the **unify** command.

Reviewer: ```suggestion
* Fixed a bug in the **unify** command when output path was provided empty.
```
